src/core/include/units/magnitude.h

```diff
@@ -356,12 +349,16 @@ template<typename T>
 concept Magnitude = detail::is_magnitude<T>;
 
 /**
- * @brief  Free-function access to the value of a Magnitude in a desired type.
- *
- * Can avoid the need for an unsightly `.template` keyword.
+ * @brief  The value of a Magnitude in a desired type T.
  */
-template<typename T>
-constexpr T get_value(Magnitude auto m) { return decltype(m)::template value<T>; }
+template<typename T, BasePower auto... BPs>
+  requires (std::is_floating_point_v<T> || (std::is_integral_v<T> && is_integral(magnitude<BPs...>{})))
```

Comment:
```suggestion
  requires std::is_floating_point_v<T> || (std::is_integral_v<T> && is_integral(magnitude<BPs...>{}))
```

Reply:
Thanks---found a few more opportunities for cleanup, too.